.travis.yml

```diff
@@ -24,12 +24,14 @@ before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
   - ./runflake8
   - ./runisort
+  - npm install -g jslint
 
 install:
   - pip install $DJANGO
   - python setup.py -q develop
 
 script:
+  - jslint django_netjsongraph/static/netjsongraph/js/*.js
```

Comment:
let's put this check in the `before_install` section so we avoid can let the build fail before the django dependencies are installed if any style issue is present, this way we can know much faster if there's a problem